src/App.js

```diff
@@ -1,23 +1,14 @@
-import logo from './logo.svg';
 import './App.css';
+import Main from './Components/Main/Main';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className="app">
+      <header className="App-header">Find Your Sign</header>
```

Comment:
Typically with components you will want them to be uppercase, for example: <Header/>